.github/workflows/publish-nym-vpn-app.yml

```diff
@@ -22,6 +22,11 @@ on:
         description: "nym-vpn-core release tag to use for Windows build (as it bundles the daemon)"
         required: false
         type: string
+      sign:
+        description: "Sign this release. Signing is billed per signature so be 100% sure you want a signed release (should be the case for stable rel only)"
```

Comment:
Suggestion, as this input make sense only for windows use `windows_sign` and add a "Windows only" mention in the desc